contracts/strategies/rfp-simple/RFPSimpleStrategy.sol

```diff
@@ -432,12 +432,12 @@ contract RFPSimpleStrategy is BaseStrategy, ReentrancyGuard {
         Milestone storage milestone = milestones[upcomingMilestone];
         Recipient memory recipient = _recipients[acceptedRecipientId];
 
-        // make sure has enough funds to distribute based on the proposal bid
-        if (recipient.proposalBid > poolAmount) revert NOT_ENOUGH_FUNDS();
-
         // Calculate the amount to be distributed for the milestone
         uint256 amount = (recipient.proposalBid * milestone.amountPercentage) / 1e18;
 
+        // make sure has enough funds to distribute the milestone
+        if (amount > poolAmount) revert NOT_ENOUGH_FUNDS();
```

Comment:
@thelostone-mc @codenamejason 
this check can be removed since it is redundant.

if amount > poolAmount: `poolAmount -= amount;` will revert with an underflow.